packages/algob/test/internal/deployer.ts

```diff
@@ -677,3 +678,92 @@ describe("DeployerDeployMode", () => {
 		assert.deepEqual(res.assetDef, asaDef);
 	});
 });
+describe("Helper functions", () => {
+	const env = mkEnv("network1");
+	const deployerCfg = new DeployerConfig(env, new AlgoOperatorDryRunImpl());
+	let deployer: DeployerDeployMode;
+	let alice: algosdk.Account;
+	let bob: algosdk.Account;
+
+	beforeEach(() => {
+		deployer = new DeployerDeployMode(deployerCfg);
+		alice = generateAccount();
+		bob = generateAccount();
+	});
+
+	it("Should return a transaction object based on provided execParams", () => {
```

Comment:
Don't use arrow function in unit test.
Please find the reason here: https://mochajs.org/#arrow-functions